core/common/src/Sinks.kt

```diff
@@ -275,6 +275,70 @@ public fun Sink.writeULongLe(long: ULong) {
     writeLongLe(long.toLong())
 }
 
+/**
+ * Writes four bytes of a bit representation of [float], in the big-endian order, to this sink.
+ * Bit representation of the [float] corresponds to the IEEE 754 floating-point "single format" bit layout.
+ *
+ * To obtain a bit representation, the [Float.toBits] function is used.
+ *
```

Comment:
Does it make sense to mention that these functions should be used with care when working with special values (`NaN` particularly) as bit patterns obtained for `Float.NaN` may vary depending on a platform?